masonite/view.py

```diff
@@ -127,15 +127,16 @@ def __is_expired_cache(self):
         Check if cache is expired
         """
 
-        # If is forever
-        if self.cache_time is None and self.cache_time is None:
-            return False
+        # Check if cache_for is set and configurate
+        if self.cache_time is None or self.cache_time is None and self.cache:
+            return True
 
         # By seconds
         cache_type = self.cache_type.lower()
         calc = 0
         if cache_type == "second" or cache_type == "seconds":
-            calc = 1
+            # Set time now for
+            calc = self.cache_time
         elif cache_type == "minutes" or cache_type == "minute":
             calc = 60
         elif cache_type == "hours" or cache_type == 'hour':
```

Comment:
Let's also change this to `or hour` and not `or cache_type == 'hour'` to make it look cleaner

Reply:
Ignore this!